devops/aws-codebuild/Jenkinsfile.ci

```diff
@@ -54,13 +54,13 @@ pipelineWrapper({
                         // not accurate (it concatenates them):
                         //  debian package version: <src_version>+<release_version>
                         //  debian package name:    <src_version><release_version>
-                        def sovtoken_deb_version = "0.9.25.98"
-                        def sovtokenfees_deb_version = "0.9.25.98"
+                        def sovtoken_deb_version = "0.9.3+12.58"
```

Comment:
We have different versions here

Reply:
@KitHat That is correct. Please see https://repo.sovrin.org/deb/pool/xenial/rc/s/sovtokenfees/

Reply:
yes, sovtoken and sovtokenfees got out of sync while CI/CD was being built for stable branch.